vignettes/Intro_PoolDilutionR.Rmd

```diff
@@ -280,10 +280,35 @@ Poor fits can result from a variety of issues, from instrument error, to incorre
 \
 
 ### Variable Fractionation  
-\
 
-### Known Production Rate  
-\
+The most common goal in pool dilution experiments is to estimate production
+`P` and consumption `k`, but `PoolDilutionR` can be used to optimize
+any or all of four parameters: `P`, `k`, `frac_P` (the isotopic fractionation
+value of production), and `frac_k` (fractionation of consumption).
+
+For example, we can tell `pdr_optimize` that the production and consumption
+values are fixed (i.e., known) and ask it to optimize only the fractionation rates:
+
+```{r}
+pdr_optimize_tidy(time = OneSample_dat$time_days,
+                  m = OneSample_dat$cal12CH4ml + OneSample_dat$cal13CH4ml, 
+                  n = OneSample_dat$cal13CH4ml,
+                  P = 0.1, 
+                  params_to_optimize = c("frac_P", "frac_k"),
```

Comment:
let's go ahead and specify a k here, instead of using the calculated k0

Let's specify P = 1, and k = 2

Reply:
Done!